widget/embedded/src/components/AppRouter.tsx

```diff
@@ -48,6 +50,14 @@ export function AppRouter({
   const Router = isRouterInContext ? Route : MemoryRouter;
   const blockchains = useAppStore().blockchains();
   const { canSwitchNetworkTo } = useWallets();
+  const { loadingStatus } = useMetaStore();
```

Comment:
I think you can move this line and the next line to the hook itself, as long as you only want to read the wallet from URL

Reply:
fixed